mmv1/third_party/terraform/tests/resource_compute_region_url_map_test.go.erb

```diff
@@ -1064,11 +1084,34 @@ resource "google_compute_region_url_map" "foobar" {
       }
     }
 
+    # update to be <1 second
+    timeout {
+      seconds = 0
+      nanos = 10000000 # 0.01 seconds
+    }
+
+    # update both values
+    url_rewrite {
+      path_prefix_rewrite = "foobar-2"
+      host_rewrite = "fizzbuzz-2"
```

Comment:
Similar comment regarding prefix and host rewrites

Reply:
Thanks! I've updated all the instances of these fields across all the url map resources' tests. As the tests have passed I'll go ahead and merge.